modules/tinymce/src/themes/silver/main/ts/api/Events.ts

```diff
@@ -39,6 +39,26 @@ const fireToggleToolbarDrawer = (editor: Editor, state: boolean): void => {
   editor.dispatch('ToggleToolbarDrawer', { state });
 };
 
+const fireStylesTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('StylesTextUpdate', data);
+};
+
+const fireAlignTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('AlignTextUpdate', data);
+};
+
+const fireFontSizeTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('FontSizeTextUpdate', data);
+};
+
+const fireBlocksTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('BlocksTextUpdate', data);
+};
+
+const fireFontFamilyTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('FontFamilyTextUpdate', data);
+};
+
```

Comment:
Does this cover all of the bespokes? We're effectively making an API here, I can imagine some integrators will be happy to see this information broadcast, so we need to make sure it's right the first time.

Reply:
Do we need one for `FontSizeInput`? https://github.com/tinymce/tinymce/blob/82f8b3cd850d5ff58f4b5f327f56d47773743a80/modules/tinymce/src/themes/silver/main/ts/ui/toolbar/Integration.ts#L118

Reply:
Yes `fontsizeinput` looks to be the only bespoke button I've missed, I've added that. I've also added tests for all the bespoke text update events.